### PR TITLE
STACK-61-filter

### PR DESCRIPTION
**Pull request by MarMil1**
Implemented filters and added style to questions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
 		"compile:vue": "vue-cli-service build --mode=development --inline-vue",
 		"compile:extension": "tsc -p ./",
 		"lint": "eslint src --ext ts",
-		"watch": "tsc -watch -p ./",
+		"watch": "vue-cli-service build --watch",
```
Comment (dbergunder) [inline package.json:31]: 👍

File: src/components/Questions.vue [hunk 2]
```diff
@@ -15,7 +15,55 @@
                 </tr>
             </tbody>
             </template>
-        </v-simple-table>
```
Comment (dbergunder) [inline src/components/Questions.vue:18]: Go ahead and remove the refactored code, no need to comment it out.  We have it in github commit history if we ever need to reference or roll back.

File: src/main.ts [hunk 3]
```diff
@@ -7,8 +7,9 @@ Vue.use(Vuetify);
 
 let v = new Vue({
     el: "#app",
+    vuetify: new Vuetify(),
```
Comment (dbergunder) [inline src/main.ts:10]: This should have been working from line 6 `Vue.use(Vuetify)` was there a conflict?

File: src/main.ts [hunk 4]
```diff
@@ -7,8 +7,9 @@ Vue.use(Vuetify);
...
     template: `
-    <div>
+    <div data-app>
```
Comment (dbergunder) [inline src/main.ts:12]: If you need to add a javascript data attribute at the root level it should probably go here: https://github.com/SOinIDE-SE491/foundation/blob/master/src-ext/extension.ts#L65

File: src/components/Questions.vue (hunk 2 above)
Comment (MarMil1) [reply src/components/Questions.vue:18]: Removed </v-simple-table>

File: src/main.ts (hunk 3 above)
Comment (MarMil1) [reply src/main.ts:10]: There was no conflict but when I tried using v-select from Vuetify but it didn't work so I did some research and figured I could add Vuetify instance and it worked after I did. The error was 'Cannot read property t of undefined'.

File: src/main.ts (hunk 4 above)
Comment (MarMil1) [reply src/main.ts:12]: I tried putting data-app in extension.ts but that didn't work.

File: src/main.ts (hunk 3 above)
Comment (dbergunder) [reply src/main.ts:10]: Cool, then I think we can remove the use declaration on line 6.

File: src/main.ts (hunk 4 above)
Comment (dbergunder) [reply src/main.ts:12]: What's it used for?

File: src/main.ts (hunk 3 above)
Comment (MarMil1) [reply src/main.ts:10]: I just tried removing Vue.use(Vuetify) and it's not working without it. It seems it needs Vue.use(Vuetify) somehow.

File: src/main.ts (hunk 4 above)
Comment (MarMil1) [reply src/main.ts:12]: I used it for the drop down menu. Somehow v-select was not working before.

Comment (dbergunder) [reply src/main.ts:12]: That is really odd. The only thing I found relating to it is this old thread: https://forum.vuejs.org/t/vuetify-data-app-true-and-problems-rendering-v-dialog-in-unit-tests/27495/9

Whatever, if we need it so be it.

Comment (MarMil1) [reply src/main.ts:12]: Yeah, it's strange. Maybe because of the webview.

**Pull request merged**